channeldb/graph.go

```diff
@@ -648,20 +646,7 @@ func (c *ChannelGraph) BatchedAddChannelEdge(edge *ChannelEdgeInfo) error {
 // supports. The chanPoint and chanID are used to uniquely identify the edge
 // globally within the database.
 func (c *ChannelGraph) AddChannelEdge(edge *ChannelEdgeInfo) error {
-	c.cacheMu.Lock()
-	defer c.cacheMu.Unlock()
-
-	err := kvdb.Update(c.db, func(tx kvdb.RwTx) error {
-		return c.addChannelEdge(tx, edge)
-	}, func() {})
-	if err != nil {
-		return err
-	}
-
-	c.rejectCache.remove(edge.ChannelID)
-	c.chanCache.remove(edge.ChannelID)
-
-	return nil
+	return c.BatchedAddChannelEdge(edge)
```

Comment:
is this change to be reverted or no? If makes sense to _only_ have `AddChannelEdge` and let that be the batched version?